tbx/static_src/sass/components/navigation/_primary-nav-mobile.scss

```diff
@@ -25,69 +29,35 @@ Styles for the top level of the navigation at mobile
         opacity: 1;
     }
 
-    // Top level menu unordered list
     &__list {
         display: flex;
         flex-direction: column;
     }
 
-    // top level menu list items
     &__item {
-        font-weight: $weight--semibold;
         position: relative;
         width: 100%;
     }
 
-    // top level menu links
     &__link {
```

Comment:
Suggestion: Shall we add the underline back in when viewing in high contrast mode?

Reply:
good one - added for both mobile and desktop items